perfscale_regression_ci/scripts/scalability/utils/cleanup.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
+
+import time
+import sys
+import ocp_utils 
+
+def delete_all_namespaces(job):
+    # make sure all namespaces are gone
+    print("job type " + str(job))
+    if job.lower() == "pod-density":
+        job = "node-density"
+    ocp_utils.run("oc delete ns --wait=false -l kube-burner-job=" + job)
+    wait_for_all_deleted_ns(job)
+
+def wait_for_all_deleted_ns(job, wait_num=300):
```

Comment:
this is a duplicate function of: https://github.com/openshift/svt/blob/7364b2b76b8ddc23fd74ca11b3c6c9f81ddeddcb/perfscale_regression_ci/scripts/common.sh#L25